wp-content/themes/newspack-theme/acf-json/group_622a69c137333.json

```diff
@@ -48,73 +29,111 @@
-            "name": "broadcast_date",
-            "type": "clone",
+            "key": "field_623cd5687a201",
+            "label": "Segments List",
+            "name": "segments_list",
+            "type": "relationship",
             "instructions": "",
             "required": 0,
-            "conditional_logic": 0,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_622a69fd0dfdf",
+                        "operator": "==",
+                        "value": "program-episode"
+                    }
+                ]
+            ],
             "wrapper": {
                 "width": "",
                 "class": "",
                 "id": ""
             },
-            "clone": [
-                "field_622111ffbae7d"
+            "post_type": [
+                "attachment"
+            ],
+            "taxonomy": "",
+            "filters": [
+                "search"
             ],
-            "display": "seamless",
-            "layout": "block",
-            "prefix_label": 0,
-            "prefix_name": 0
+            "elements": "",
+            "min": "",
+            "max": 20,
+            "return_format": "object"
         },
         {
-            "key": "field_622a6ac11faf8",
-            "label": "Expiration Date",
-            "name": "expiration_date",
-            "type": "date_picker",
+            "key": "field_622a6a5bbd407",
+            "label": "Program",
+            "name": "program",
+            "type": "taxonomy",
             "instructions": "",
             "required": 0,
-            "conditional_logic": 0,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_622a69fd0dfdf",
+                        "operator": "!=",
+                        "value": "standard"
+                    }
+                ]
+            ],
             "wrapper": {
                 "width": "",
                 "class": "",
                 "id": ""
             },
-            "display_format": "F j, Y",
-            "return_format": "Ymd",
-            "first_day": 1
+            "taxonomy": "program",
+            "field_type": "multi_select",
+            "allow_null": 1,
+            "add_term": 0,
+            "save_terms": 0,
+            "load_terms": 0,
+            "return_format": "object",
+            "multiple": 0
         },
         {
-            "key": "field_622a6aed78236",
+            "key": "field_62425676f5ee3",
             "label": "Audio Byline",
             "name": "audio_byline",
             "type": "taxonomy",
             "instructions": "",
             "required": 0,
-            "conditional_logic": 0,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_622a69fd0dfdf",
+                        "operator": "!=",
+                        "value": "program-segment"
```

Comment:
Don't we want this on segment audio? Shouldn't this be `!=` to `standard`?

Reply:
Fixed!